types/events.go

```diff
@@ -136,7 +136,7 @@ type EventDataNewBlock struct {
 }
 
 // TypeTag implements the required method of jsontypes.Tagged.
-func (EventDataNewBlock) TypeTag() string { return "tendermint/event/NewBlock" }
+func (EventDataNewBlock) TypeTag() string { return "tendermint/event/NewBlock_new" }
```

Comment:
i assume there is no interface currently using the `NewBlock_new` defined by tm0.35 correct?